rjsone/def.bzl

```diff
@@ -28,6 +28,20 @@ def _keyed_contexts_to_args(keyed_contexts):
 
     return args
 
+def _keyed_raw_contexts_to_args(keyed_raw_contexts):
+    args = []
+    for context_target, context_key in keyed_raw_contexts.items():
+        files = context_target.files.to_list()
+        if len(files) == 0:
+            fail("Target %s produces no files" % context_target.label, attr = "keyed_raw_contexts")
+        elif len(files) == 1:
+            args.append("%s::%s" % (context_key, files[0].path))
+        else:
+            args.append("%s::.." % context_key)
```

Comment:
The only difference with the function above is that we use double columns here. As per https://github.com/wryun/rjsone